firmware/flash-dfu-full.sh

```diff
@@ -9,4 +13,19 @@ while :; do
   lsusb|grep -q 1915:521f && break
   sleep 1
 done
-nrfutil device program --firmware objects/dfu-full.zip --traits nordicDfu
+
+device_type=ultra
```

Comment:
You have device type hardcoded here

Reply:
Check the line below :)

Reply:
defaults to ultra, if a lite is found it sets device_type to lite

Reply:
Oh, sorry, my mistake